site/content/docs/heartbeat-checks/_index.md

```diff
@@ -0,0 +1,99 @@
+
+Use heartbeat checks to monitor backup jobs, data imports, and other recurring jobs or scripts.
+
+Here is an example of how to have a Heroku job send a ping to a Checkly Heartbeat check.
+{{< tabs "Heroku example" >}}
+{{< tab "BASH" >}}
+```BASH
+curl -m 5 --retry 3 https://api.checklyhq.com/heartbeats/ping/bcd964a7-6f15-49a5-bac1-4be8059670ec;
+```
+{{< /tab >}}
+{{< /tabs >}}
+Note the use of the retry option. We recommend always using retries when available to avoid false alarms due to temporary network issues or other disruptions.
+
+## Creating a heartbeat check
+
+To create a new Heartbeat check, click the `+` icon on the sidebar & select **Heartbeat check**.
+
+Creating a heartbeat check is quick and easy; the check requires a name and the period and grace settings defined. These can all be changed later on. Optionally, you can customize tags or [alert settings](/docs/alerting).
+
+Once you have created your check, the service or host you want to monitor needs to be configured to send a request to the ping URL. When creating or editing the check you can find code examples for how to send requests using Javascript, Python or in Bash in the quickstart section.
+
+## Check breakdown
+
+### Name and tag
+The check name is used for identifying the check in the list of heartbeat checks and in alert messages. Using a clear and meaningful name will help team members identify the check and can help reduce the reaction time when alerts are triggered. 
+Tags are used to create meaningful distinctions between check groups, making it easy to filter out selections of checks. 
+![name and tag](/docs/images/heartbeat-checks/getting-started-name-and-tag.png)
+
+### Ping URL
+The URL on which the check is listening for pings. The job or task monitored should make an HTTP request to the ping URL once per the period configured.
+![ping url](/docs/images/heartbeat-checks/getting-started-ping-url.png)
+
+### Period and Grace
+**Period** defines how often you expect a ping to the ping URL. 
+
+**Grace** is the time Checkly will wait before triggering any alerts when a ping does not arrive within the set period. E.g., if you have a check that expects a ping every 60 minutes, with a grace of 10 minutes, no alarms would trigger until 70 minutes after the latest ping.
+
+Use grace to compensate for variance in your jobs.
+![period and grace](/docs/images/heartbeat-checks/getting-started-period-and-grace.png)
+
+### Timer
+The check timer starts when it receives its first ping and will reset after each ping. 
+If you have a check that expects a ping every 60 minutes starting at 09:30, and it receives a ping at 10:00, it will reset the timer to expect a ping before 11:00. If the check does not receive a ping before 11:00 plus any configured grace period it will trigger any configured alerts.
+> When a check is deactivated, and activated again the timer needs to be pinged to start it, just as when first creating a check.
+
+### Ping now
```

Comment:
We could also briefly show the rest Ping Now from the heartbeat dashboard quick menu
![image](https://github.com/checkly/docs.checklyhq.com/assets/25155364/195825cb-2104-4fac-9917-31e44eef3742)